exercice2.php

```diff
@@ -2,6 +2,35 @@
 
 require_once('vendor/autoload.php');
 
-use Ipssi\Evaluation\Useless;
+use IPSSI\POO\Position;
+use IPSSI\POO\Forme;
+use IPSSI\POO\Image;
+use IPSSI\POO\Texte;
+use IPSSI\POO\Couleur;
+use IPSSI\POO\View\Sheet;
 
-new Useless(); // Ceci ne sert à rien
+
+$p1 = new Position(
+    '18',
+    '10'
+);
+
+$f1 = new Forme(
```

Comment:
chaque élément doit avoir une position
```suggestion
$t1 = new Texte($p1, 'Hello');
```